wsagent/che-core-api-git/pom.xml

```diff
@@ -87,6 +87,10 @@
             <artifactId>che-core-commons-lang</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>wsagent-local</artifactId>
```

Comment:
che-core-api-git will be used for codenvy application also. Are you sure that it is OK that wsagent-local will be used there?